src/servers/tests/http/http_handler_test.rs

```diff
@@ -135,3 +135,18 @@ fn create_query() -> Query<http_handler::SqlQuery> {
         database: None,
     })
 }
+
+/// Currently the payload of response should be simply an empty json "{}";
+#[tokio::test]
+async fn test_health() {
```

Comment:
https://github.com/GreptimeTeam/greptimedb/pull/590 add an integration test module that covers serval HTTP APIs, maybe this one can also be included there.

Reply:
I will add a test case later. Actually I tried to find where to test http api but missed that part, haha